### PR TITLE
add a specific reward for solving a mission (without stars)

### PR DESCRIPTION
**Pull request by privat**
@ppepos proposes that stars are extra and that a reward is given when a mission is solved.

This PR need an improved UI on the mission page as the basic reward is not displayed.

### CODE REVIEW

**Comment by ppepos** [comment]
+1

File: src/model/missions.nit [hunk 1]
```diff
@@ -37,10 +37,14 @@ class Mission
 	var parents = new Array[String]
 	var stars = new Array[MissionStar]
 
+	# Number of points to solve the mission (excluding stars)
+	var solve_reward: Int = 1 is writable
```
Comment (Morriar) [inline src/model/missions.nit:41]: Why not `reward`?

Comment (privat) [reply src/model/missions.nit:41]: `reward` already exist and means _total reward of the mission incl stars_. Moreover it is already used by angular, so renaming will be more complex.

Comment (Morriar) [reply src/model/missions.nit:41]: Alright then

**Comment by Morriar** [comment]
+1

**Pull request merged**